pxtrunner/renderer.ts

```diff
@@ -831,6 +835,33 @@ namespace pxt.runner {
         })
     }
 
+    function renderDirectPython(options?: ClientRenderOptions) {
+        // Highlight python snippets written with the ```python
+        // language tag (as opposed to the ```spy tag, see renderStaticPythonAsync for that)
```

Comment:
does checkdocs need to be updated to handle these snippets?

Reply:
Good question

Reply:
Updating checkdocs is non-obvious, I'm going to hold this PR until I can talk to someone in person that knows more

Reply:
Updated check docs. It can now correctly detect if there are errors in the python snippets. Good catch